pkg/core/state/notification_event.go

```diff
@@ -4,7 +4,6 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
```

Comment:
Shouldn't happen.

Reply:
Not sure what the request is here. Comparing it to master there are no spaces there either, just new lines (`0a`)
https://github.com/nspcc-dev/neo-go/blob/2b758c53ebadd2884492392d210ad823ff9996df/pkg/core/state/notification_event.go#L6-L8

Maybe one of my old commits had it slipped in?

Reply:
Yes, one commit removes it and another one adds it back.